pkgs/applications/graphics/mcomix3/default.nix

```diff
@@ -0,0 +1,98 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, python3
+, python3Packages
```

Comment:
Use either `python3Packages.python` with `python3Packages.foomodule`, or `python3` with `python3.pkgs.foomodule`. Mixing the two styles can lead to inconsistencies when one is overridden.